playbooks/roles/handle-nodes-on-openstack/tasks/create_on_openstack.yml

```diff
@@ -37,7 +37,7 @@
         hosts:
           {{ servername }}:
             ansible_host: {{ floating_ip }}
-            {% if ansible_user|default(False) %}ansible_user: {{ ansible_user }}{% endif %}
+            ansible_user: {{ ssh_user|default('root') }}
```

Comment:
Please mention in the commit msg why this is useful/needed

Reply:
sorry about that, had to send the patch fast as it was late in the night, my brain was not able to write more than this  :)